doc/man3/RAND_bytes.pod

```diff
@@ -44,10 +44,21 @@ ignored.
 
 =head1 NOTES
 
-Always check the error return value of RAND_bytes() and
-RAND_priv_bytes() and do not take randomness for granted: an error occurs
-if the CSPRNG has not been seeded with enough randomness to ensure an
-unpredictable byte sequence.
+By default, the OpenSSL CSPRNG supports a security level of 256 bits, provided it
+was able to seed itself with 256 bits of entropy from a trusted entropy source.
```

Comment:
`d/itself with 256 bits of entropy/`

It grabs 384 bits, so best to not say how much :)

Reply:
Fixed in 68d8704e8c2ec5c92ce769592babcb74e31c27e3.